backend/app/Http/Controllers/QuestionController.php

```diff
@@ -2,31 +2,31 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Category;
 use App\Models\Question;
-use App\Http\Requests\StoreQuestionRequest;
-use App\Http\Requests\UpdateQuestionRequest;
 use Illuminate\Http\Request;
 
 class QuestionController extends Controller
 {
+    const STORAGE_PATH = 'questions';
     /**
      * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Category $category)
     {
-        //
+        return $this->showAll(Question::all()->where('category_id', $category->id));
```

Comment:
Create a relationship between Question and category, so that you can use the `with()` method useful when retrieving large datasets because it has a eager loading feature

Reply:
https://laravel.com/docs/8.x/eloquent-relationships